src/SFML/PCH.hpp

```diff
@@ -0,0 +1,62 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2021 Laurent Gomila (laurent@sfml-dev.org)
```

Comment:
Should be 2022.
Likely also affects the source from which the license header was copy-pasted.